Cargo.toml

```diff
@@ -8,5 +8,10 @@ readme = "README.md"
 license = "MIT/Apache-2.0"
 
 [dependencies]
-log = "0.*"
 bit_field = "0.9.0"
+
+[target.'cfg(not(test))'.dependencies]
+log = "0.4.5"
+
+[target.'cfg(test)'.dependencies]
+log = { version = "0.4.5", features = ["std"] }
```

Comment:
This seems to break usage within a kernel by turning the `std` feature on when used as a dependency, but I'm pretty sure `cfg(test)` should be false. Not sure what's happening